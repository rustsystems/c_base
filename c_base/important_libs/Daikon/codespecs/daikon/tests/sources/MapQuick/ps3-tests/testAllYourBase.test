#Test the All Your Base graph
CreateGraph fun

CreateNode You 1
CreateNode All 1
CreateNode belong 1
CreateNode to 1
CreateNode us 1
CreateNode chance 1
CreateNode make 1
CreateNode your 1
CreateNode base 1
CreateNode are 1
CreateNode have 1
CreateNode no 1
CreateNode time 1
CreateNode survive 1

AddNode fun You
AddNode fun All
AddNode fun belong
AddNode fun to
AddNode fun us
AddNode fun chance
AddNode fun make
AddNode fun your
AddNode fun base
AddNode fun are
AddNode fun have
AddNode fun no
AddNode fun time
AddNode fun survive

AddEdge fun chance base
AddEdge fun your base
AddEdge fun are belong
AddEdge fun make your
AddEdge fun to us
AddEdge fun You have
AddEdge fun your time
AddEdge fun have no
AddEdge fun no chance
AddEdge fun chance to
AddEdge fun base are
AddEdge fun to survive
AddEdge fun All your
AddEdge fun survive make
AddEdge fun belong to

FindPath fun All -> us
FindPath fun You -> time


