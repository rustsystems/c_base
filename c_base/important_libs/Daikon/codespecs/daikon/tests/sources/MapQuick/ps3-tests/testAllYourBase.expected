#Test the All Your Base graph
created graph fun

created node You with cost 1
created node All with cost 1
created node belong with cost 1
created node to with cost 1
created node us with cost 1
created node chance with cost 1
created node make with cost 1
created node your with cost 1
created node base with cost 1
created node are with cost 1
created node have with cost 1
created node no with cost 1
created node time with cost 1
created node survive with cost 1

added node You to fun
added node All to fun
added node belong to fun
added node to to fun
added node us to fun
added node chance to fun
added node make to fun
added node your to fun
added node base to fun
added node are to fun
added node have to fun
added node no to fun
added node time to fun
added node survive to fun

added edge from chance to base in fun
added edge from your to base in fun
added edge from are to belong in fun
added edge from make to your in fun
added edge from to to us in fun
added edge from You to have in fun
added edge from your to time in fun
added edge from have to no in fun
added edge from no to chance in fun
added edge from chance to to in fun
added edge from base to are in fun
added edge from to to survive in fun
added edge from All to your in fun
added edge from survive to make in fun
added edge from belong to to in fun

shortest path in fun: All your base are belong to us
shortest path in fun: You have no chance to survive make your time


